src/v3/src/components/Images/MobileDeviceIcon.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { FunctionComponent, h } from 'preact';
+
+import { loc } from '../../util/locUtil';
+
+export const MobileDeviceIcon: FunctionComponent = () => (
+  <svg
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    role="img"
+    aria-hidden="true"
+  >
+    <title id="mobile-icon">
+      {loc('icon.title.browser', 'login')}
```

Comment:
Should this be `icon.title.mobileDevice`

Reply:
`icon.title.mobileDevice` is used for the phone image in the Okta Verify push code flow.  The "Mobile browser" key needed for this icon is actually missing so I will add it